client/src/components/AttendeeInput.tsx

```diff
@@ -33,44 +32,24 @@ export default function AttendeeInput({ id, onChange, value, type }: AttendeeInp
 
   const handleSelectionChange = (_: React.SyntheticEvent, newValue: Array<string | IPeopleInformation>) => {
     const emails = newValue.map((option) => (typeof option === 'object' && option.email ? option.email : (option as string)));
-    const filteredEmails = emails.filter((email) => emails.indexOf(email) === emails.lastIndexOf(email));
-    if (filteredEmails.length > 0) {
-      const lastValue = filteredEmails[filteredEmails.length - 1].trim();
-      if (isEmailValid(lastValue)) {
-        onChange(id, filteredEmails);
-        setTextInput('');
-      } else {
-        toast.error('Invalid email entered');
-      }
-    } else {
-      onChange(id, filteredEmails);
-      setTextInput('');
+    const filteredEmails = emails
+      .join(' ')
+      .split(/\s+/)
+      .map((email) => email.trim())
+      .filter((email) => email !== '');
+    const uniqueEmails = [...new Set(filteredEmails)];
+    const validEmails: string[] = [],
+      invalidEmails: string[] = [];
+    uniqueEmails.forEach((email) => {
+      isEmailValid(email) ? validEmails.push(email) : invalidEmails.push(email);
+    });
+    invalidEmails.length == 1 ? toast.error('Invalid email entered.') : invalidEmails.length > 1 ? toast.error('Invalid emails entered.') : null;
```

Comment:
Instead of showing 2 different toasts, it could be re-written as "Invalid email(s) entered"